core/store/migrate/migrations/0188_create_automation_upkeep_state.sql

```diff
@@ -0,0 +1,21 @@
+-- +goose Up
+
+CREATE TABLE evm_upkeep_states (
+    work_id TEXT PRIMARY KEY,
+    evm_chain_id NUMERIC NOT NULL,
+    upkeep_id BYTEA NOT NULL,
+    completion_state SMALLINT NOT NULL,
+    ineligibility_reason NUMERIC NOT NULL,
```

Comment:
Why is this `NUMERIC` here but `uint8` in code? Please make all DB types match code types.